CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [9.0.0]
+
+### Changed
+
+- **BREAKING:** Format the changelog using Prettier. ([#361](https://github.com/MetaMask/keyring-api/pull/361))
```

Comment:
I upgraded the release to a major version bump based on this entry being a breaking change. Feel free to correct me if this wasn't the team's intention.

Reply:
@owencraston How to categorize this is up to the team, but my intuition is that the changelog is a user-facing component of the package. If the application of prettier rules retroactively alters the changelog and/or enforces a new format going forward, I think it's a noteworthy update. I assume that's why it was categorized as a breaking change in the PR.

Reply:
Since it doesn't impact the users, it shouldn't be a major release.

The changelog was already formatted using Prettier, but it was done manually in the release branches and enforced by the CI.

Reply:
Ah so there's no impact to the changelog -- makes sense not to see it as a breaking change. 

Reverted to patch release here: https://github.com/MetaMask/keyring-api/pull/360/commits/36c7f6118fb4c671b26823336845d1488dd864bc

Also removed the entry for the CI update as irrelevant to consumers.